website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -0,0 +1,41 @@
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'typescript-eslint-parser/package.json';
+
+const ID = 'typescript-eslint-parser';
+const name = 'typescript-eslint-parser';
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: name,
+  version: pkg.version,
+  homepage: pkg.homepage,
+  locationProps: new Set(['loc', 'start', 'end', 'range']),
+
+  loadParser(callback) {
+    require(['typescript-eslint-parser'], typescriptEslint => callback(typescriptEslint));
```

Comment:
Nit: no need to create extra arrow function here, you can use just `require(['typescript-eslint-parser'], callback)` as we do in many other places.